### PR TITLE
FIX Fix link for managing roles

### PR DESCRIPTION
**Pull request by GuySartorelli**
## Description

Fix "manage roles" link to correctly link to the roles tab

## Manual testing steps

1. Within CMS, select Security
1. Go to "Roles" tab and create a role
2. Go back to Security
3. Go to "Groups" tab
4. Select any group
5. Go to "Roles" tab (within group edit form)
6. click "Manage Roles" link (this should take you to the main roles section)

## Issues

- https://github.com/silverstripe/silverstripe-admin/issues/1700

## Pull request checklist

- [x] The target branch is correct
    - See [picking the right version](https://docs.silverstripe.org/en/contributing/code/#picking-the-right-version)
- [x] All commits are relevant to the purpose of the PR (e.g. no debug statements, unrelated refactoring, or arbitrary linting)
    - Small amounts of additional linting are usually okay, but if it makes it hard to concentrate on the relevant changes, ask for the unrelated changes to be reverted, and submitted as a separate PR.
- [x] The commit messages follow our [commit message guidelines](https://docs.silverstripe.org/en/contributing/code/#commit-messages)
- [x] The PR follows our [contribution guidelines](https://docs.silverstripe.org/en/contributing/code/)
- [x] Code changes follow our [coding conventions](https://docs.silverstripe.org/en/contributing/coding_conventions/)
- [x] This change is covered with tests (behat tests will be added in an admin PR)
- [x] CI is green

### CODE REVIEW

File: src/Security/Group.php [hunk 1]
```diff
@@ -234,8 +234,8 @@ class_exists(SecurityAdmin::class)
                     ) . '<br />' .
                     sprintf(
                         '<a href="%s" class="add-role">%s</a>',
-                        SecurityAdmin::singleton()->Link('show/root#Root_Roles'),
-                        _t('SilverStripe\\Security\\Group.RolesAddEditLink', 'Manage roles')
+                        SecurityAdmin::singleton()->Link('roles'),
+                        _t(__CLASS__ . '.RolesAddEditLink', 'Manage roles')
```
Comment (GuySartorelli) [inline src/Security/Group.php:238]: Changing `'SilverStripe\\Security\\Group.RolesAddEditLink'` to `__CLASS__ . '.RolesAddEditLink'` is an unrelated refactoring - figured I should tidy that up while I'm in here.

**Review by michalkleiner — APPROVED** [review]

**Review by emteknetnz — APPROVED** [review]
Tested locally, works correctly
